bin/verilator

```diff
@@ -358,6 +358,7 @@ detailed descriptions of these arguments.
     --output-split-cfuncs <statements>   Split model functions
     --output-split-ctrace <statements>   Split tracing functions
      -P                         Disable line numbers and blanks with -E
+    --verilation-threads <jobs> Run Verilator on <jobs> threads when possible
```

Comment:
Can be just call this `-j`?

Reply:
More precisely: use the value of -j

Reply:
I wanted to use ``-j`` as this option name, but I saw it is already taken by some other option. If it is ok to use value of ``-j`` as number of threads for verilation and build, than I'm ok with it.

Changed to use value of ``-j`` as number of threads.

Reply:
Hmm.  I suspect the user will use -j when they want things faster and are using many cores, so having -j control this makes sense.  The problem is it also controls make jobs, and that might need to be set differently if e.g. using a distcc distribution tool.

I'd suggest make a pull request to add --build-jobs which does what -j does now, then have -j also set build jobs.

Then this pull would add --verilation-jobs and -j also sets that.  We'd document that -j sets both, and for finer control use the separate flags after the -j.

Reply:
Just a note that when `--build` was added, we discussed that the actual build process is up to Verilator at that point without any implementation commitments (hence we don't support `--build` with CMake, and currently we use Make), so I don't think using distcc is in scope for `--build`. We can expect advanced users to build externally.

Having said that I am happy with the `-j` `--verilation-jobs` `--build-jobs` split.